### PR TITLE
Replace TypeScript with Python

### PR DESCRIPTION
**Pull request by amorriscode**
After having a lot of audio related issues with TypeScript on the terminal, we have decided to replace it with Python. This will make our lives a lot better.

This PR:
- [x] Removes all TypeScript code and tooling
- [x] Creates a quick scaffold for the Python code (which only handles the song/transport for now)

Future PRs:
- [ ] Render out some frets
- [ ] Parse MIDI into memory

I'm not a Pythonista so the structure might not be appropriate. Just did some quick Googling and made assumptions.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,10 +1,5 @@
...
-/lib
-/package-lock.json
 /tmp
-node_modules
 songs
-.DS_Store
\ No newline at end of file
+.DS_Store
+th-env
+__pycache__
```
Comment (hyshka) [inline .gitignore]: We may want to add `*.pyc` here as well.

File: README.md [hunk 2]
```diff
@@ -1,32 +1,49 @@
...
-  $ terminal-hero COMMAND
-...
+* [Songs](#songs)
+
+## Getting Started
+
+1. Create and activate a `virtualenv` with python:
+
+```bash
+python3 -m virtualenv th-env
```
Comment (hyshka) [inline README.md]: I can set up a makefile to make this process a bit quicker in the future.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+PyOpenAl==0.7.11a1
+PyOgg==0.6.13a1
+argparse==1.4.0
```
Comment (hyshka) [inline requirements.txt]: While we're developing we can do stuff like `argparse==1.4.*` or if we're really trusting `argparse==1.*` and we won't have to update this file as often to get package updates. There's other syntax that is equivalent such as `argparse>=1.4`.

**Review by hyshka — APPROVED** [review]
There's not very much code there to get audio playing! Pretty sweet.

My comments are all just QoL stuff that we can play around with as we go.

File: README.md (hunk 2 above)
Comment (amorriscode) [reply README.md]: Yeah I was being lazy haha.

**Pull request merged**